### ISSUE TITLE
Reference filter bar should be sticky

### ISSUE THREAD

**Issue by jcubic** [LIPS-scheme/lips#364]
If you search in Algolia and land on the reference page, you can't use the filter. It should be always visible (use `position: sticky`)

Also, maybe it would be nice to improve the design of the search bar.

**Comment by vidhihattar** [comment]
Hi, I would like to work on this issue. Could you please assign it to me?

**Comment by jcubic** [comment]
Open Source project, don't use assignments on GitHub. Especially for non-core contributors.

**Comment by vidhihattar** [comment]
Thank you for letting me know. I saw other people asking to be assigned the
issues in OS projects so I thought that's how it is supposed to be.

**Comment by jcubic** [comment]
I have no idea why people keep asking this.

**Comment by vidhihattar** [comment]
Sorry to bother you but I wanted to ask if this is an error on my end?
please see the image attached. When I run the make command it says that
there is an issue in the makefile. I searched online and it said that I
needed to make changes in the makefile so I wanted to confirm if this is an
issue on my system.

**Comment by jcubic** [comment]
No, sorry the Makefile is only for source code. I need to update the Contribution guide for the documentation that was recently created. To work on the docs locally you need:

```bash
cd docs
npm install
npm run start
```

**Comment by jcubic** [comment]
Also note that if you reply to an issue with an email, attachments are removed. To add an image you need to use the GitHub interface.

**Comment by jcubic** [comment]
And when you will create a branch use master, so the changes will be instantly visible on the website.

**Comment by vidhihattar** [comment]
Got it, thanks for the help!
